apps/core/src/pages/privacy-policy/index.tsx

```diff
@@ -0,0 +1,351 @@
+        variant: "filled",
+        color: theme.primaryColor
+      }).background,
+      color: theme.white
+    }
+  }
+}));
+
+const breadcrumbs = [
+  { title: "Privacy & Terms ", href: "#" },
+  { title: "Privacy policy", href: "#" }
+].map((item, index) => (
+  <Anchor href={item.href} key={index}>
+    {item.title}
+  </Anchor>
+));
+
+const linksMockdata = [
+  { title: "The Scope of This Privacy Policy", id: "privacyScope" },
+  { title: "Information we collect", id: "informationCollection" },
+  {
+    title: "How We Process Your Information, and the Laws That Support It",
+    id: "informationProcessing"
+  },
+  { title: "Information we share", id: "information-sharing" },
+  { title: "Data Retention", id: "data-retention" },
+  { title: "Security", id: "security" },
+  {
+    title: "Modifications to this privacy policy",
+    id: "privacyModification"
+  },
+  { title: "Our global operations", id: "global-ops" },
+  { title: "Your Rights", id: "rights" },
+  { title: "Data Protection Authority", id: "data-auth" },
+  { title: "Limitation of liability", id: "limitation" },
+  {
+    title: "Member account, Password, and Security",
+    id: "member"
+  },
+  { title: "Use of services", id: "service-use" },
+  { title: "Privacy Policy", id: "privacy" }
+];
+
+const PrivacyPage = () => {
+  const dispatch = useAppDispatch();
+
+  dispatch(uiSliceActions.setSidebarVisibility(false));
+  const { classes, cx } = useStyles();
+  const [active, setActive] = useState("Content");
+  const [activeLink, setActiveLink] = useState("privacyScope");
+
+  const links = linksMockdata.map(link => (
+    <a
+      className={cx(classes.link, {
+        [classes.linkActive]: activeLink === link.id
+      })}
+      href={`#${link.id}`}
+      onClick={() => {
+        setActiveLink(link.id);
+      }}
+      key={link.id}
+    >
+      {link.title}
+    </a>
+  ));
+
+  const headerRef = useRef<HTMLDivElement>(null);
+  const sectionWrapperRef = useRef<HTMLDivElement>(null);
+
+  const pageSections = [
+    { pageId: "PrivacyScope", pageRef: useRef<HTMLDivElement>(null) },
+    { pageId: "PrivacyModification", pageRef: useRef<HTMLDivElement>(null) },
+    { pageId: "InformationCollection", pageRef: useRef<HTMLDivElement>(null) },
+    { pageId: "InformationSharing", pageRef: useRef<HTMLDivElement>(null) },
+    { pageId: "InformationProcessing", pageRef: useRef<HTMLDivElement>(null) },
+    { pageId: "GlobalOperations", pageRef: useRef<HTMLDivElement>(null) },
+    { pageId: "Security", pageRef: useRef<HTMLDivElement>(null) },
+    { pageId: "YourRights", pageRef: useRef<HTMLDivElement>(null) },
+    { pageId: "DataRetention", pageRef: useRef<HTMLDivElement>(null) },
+    { pageId: "DataAuthority", pageRef: useRef<HTMLDivElement>(null) }
+  ];
+
+  // this is supposed to scroll the page to the desired section when the link on the sidebar is clicked but adjust it into the users view by 400px. It does not work properly at the moment
+  useEffect(() => {
```

Comment:
what's not working with it?

let me know maybe I can help

Reply:
On this page, the sidebar has links that when clicked on, the other section should adjust to show what that link is tied to (in this case, the section of that page). But, currently, it adjusts to that point but the heading is hidden. So this code was supposed to make it scroll down by distance so that the title of that section is visible to the user. The code I wrote doesn't do that now.